### PR TITLE
Fb ajax

### PR DESCRIPTION
**Pull request by FadiBaker92**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -59,7 +62,10 @@ def destroy
     respond_to do |format|
       format.html { redirect_back fallback_location: root_url, notice: "Comment was successfully destroyed." }
       format.json { head :no_content }
+      format.js 
     end
+
```
Comment (ahuynh3a) [inline app/controllers/comments_controller.rb]: nitpik: Please be mindful of any empty whitespaces you might not need, and remove them.

File: app/views/follow_requests/_follow_unfollow.html.erb [hunk 2]
```diff
@@ -1,17 +1,19 @@
-<% unless sender == recipient %>
-  <% follow_request = current_user.sent_follow_requests.find_by(recipient: recipient) %>
+<div class="<%=dom_id(recipient)%>_<%=dom_id(sender)%>_form">
```
Comment (ahuynh3a) [inline app/views/follow_requests/_follow_unfollow.html.erb:1]: great job using a class here

File: app/views/follow_requests/_form.html.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<%= form_with(model: follow_request, class: "d-inline-block") do |form| %>
+<%= form_with(model: follow_request, class: "d-inline-block" local: false) do |form| %>
```
Comment (ahuynh3a) [inline app/views/follow_requests/_form.html.erb:1]: great job being mindful of when to use local: false and when to use remote:true

File: app/views/follow_requests/destroy.js.erb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+var unfollow_btn = $(".<%=dom_id(@follow_request.recipient)%>_<%=dom_id(@follow_request.sender)%>_form")
```
Comment (ahuynh3a) [inline app/views/follow_requests/destroy.js.erb]: I love that you used variables!

Comment (ahuynh3a) [inline app/views/follow_requests/destroy.js.erb]: you are forgetting a semi-colon here so your follow/unfollow button is not working

**Review by ahuynh3a — CHANGES_REQUESTED** [review]
Hey fadi! You code looks great! One thing, in your follow_request/destroy.js.erb, you are missing a semicolon after your first line of code so I am unable to destroy the follow, could you add that semicolon? I can approve after!

**Comment by FadiBaker92** [comment]
Thank you for the feedback Annie! I  added the semicolon and removed the extra white spaces.

**Review by ahuynh3a — APPROVED** [review]
Thank you for making the changes!

**Pull request merged**